shielddash/api/models.py

```diff
@@ -1,8 +1,36 @@
 from django.db import models
+from django.utils import timezone
 
 
-class Retention(models.Model):
+class Study(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(default='')
```

Comment:
@spasovski just mentioned that on IRC, I guess that's something we'll have to ask the maintainers to add optionally?